### PR TITLE
DEMO-111: [Product] - Full view

### PR DESCRIPTION
**Pull request by damianz5**
https://jira.ez.no/browse/DEMO-111

tomato soup

### CODE REVIEW

File: app/Resources/views/themes/tastefulplanet/full/product.html.twig [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+{% endblock %}
+
+{% block content %}
+    {% include '@ezdesign/parts/product_modal_dialog.html.twig' %}
+
+    <section class="content-type-product full-view">
+        <div class="breadcrumb">
+            <div class="container">
+                <a href="{{ path( ezpublish.rootLocation() ) }}">Home</a> &gt;
+                <a href="{{ path('ez_urlalias', {locationId: location.parentLocationId}) }}">{{ 'Products'|trans }}</a> &gt;
```
Comment (SylvainGuittard) [inline app/Resources/views/themes/tastefulplanet/full/product.html.twig]:
```
{{ 'Products'|trans }}
```
Not a big fan of this. We should display the name of the parent

File: web/assets/css/product.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
+.content-type-product {
+    background: #fff;
+}
+
+.content-type-product .breadcrumb {
+    color: #8D8D8D;
```
Comment (SylvainGuittard) [inline web/assets/css/product.css]: replace by `color: #8d8d8d;`

File: web/assets/css/product.css [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    line-height: 12px;
+    font-size: 14px;
+}
+
+.content-type-product .breadcrumb a {
+    color: #8D8D8D;
```
Comment (SylvainGuittard) [inline web/assets/css/product.css]: replace by `color: #8d8d8d;`

File: web/assets/css/product.css [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+.product-main {
+    padding-top: 44px;
+}
+
+.product-main .btn {
+    margin-bottom: 20px;
+    border-radius: 0;
+    text-transform: uppercase;
+    font-size: 10pt;
```
Comment (SylvainGuittard) [inline web/assets/css/product.css]: `pt`? I think it's bette to use `px`

File: web/assets/css/product.css [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+.product-main-title h2 {
+    color: #63605D;
```
Comment (SylvainGuittard) [inline web/assets/css/product.css]: please replace by `color: #63605d;`

File: web/assets/css/product.css [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+    font-weight: bold;
+}
+
+.product-main-add {
+    padding: 0;
+    margin-bottom: 10px;
+}
+
+.product-dietary-list h3 {
+    font-size: 12pt;
```
Comment (SylvainGuittard) [inline web/assets/css/product.css]: `pt`? I think it's bette to use `px`

File: web/assets/css/product.css [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+    padding: 0 7px 0 0;
+    font-size: 15px;
+    color: #63605d;
+}
+
+.product-main-price {
+    font-size: 35px;
+    font-weight: bold;
+    line-height: 48px;
+    color: #0073AE;
```
Comment (SylvainGuittard) [inline web/assets/css/product.css]: replace by `color: #0073ae;`

File: web/assets/css/product.css [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+    display: flex;
+}
+
+.product-main-serving .serving-label {
+    display: inline-block;
+}
+
+.product-main-spicy {
+    display: flex;
+    color: #63605D;
```
Comment (SylvainGuittard) [inline web/assets/css/product.css]: replace by `color: #63605d;`

**Comment by SylvainGuittard** [comment]
@damianz5 Thank you for this PR. I think it's a good start. We can also optimize the svg files.

**Comment by damianz5** [comment]
ready to review

**Pull request merged**